ocaml/xapi/importexport.ml

```diff
@@ -277,7 +277,12 @@ let remote_metadata_export_import ~__context ~rpc ~session_id ~remote_address ~r
                         )
                      )
                 )
-             );
+             )
+             with e ->
```

Comment:
This would be better as
```
    ...
  )
with Xmlrpc_client.Stunnel_connection_failed ->
  raise (Api_errors.Server_error(Api_errors.tls_connection_failed, [remote_address; !Xapi_globs.https_port]))
end;
```
There is no need for `with e -> match e with`.
Also the indentation is a bit inconsistent here at present.